versioned_docs/version-2.10/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md

```diff
@@ -286,3 +288,17 @@ To annotate a registered cluster,
 6. Click **Save**.
 
 **Result:** The annotation does not give the capabilities to the cluster, but it does indicate to Rancher that the cluster has those capabilities.
+
+## Troubleshooting
+
+This section lists some of the most common errors that may occur when importing a cluster, along with steps to troubleshoot them.
+
+### AKS
+
+#### Error: Getting static credential is not allowed because this cluster is set to disable local accounts.
+
+* This may occur if your cluster has disabled local accounts. Enable local accounts before attempting to import the cluster again:
+
+  ```
```

Comment:
to align with https://github.com/rancher/rancher-docs/pull/1590/files#r1884359891

```suggestion
  ```sh
```